migrate-from-mariadb.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Migrate Data from MariaDB to TiDB
+summary: Learn how to migrate data from MariaDB to TiDB.
+---
+
+# Migrate Data from MariaDB to TiDB
+
+This document describes how to migrate data from a MariaDB server installation to a TiDB cluster.
+
+## Prerequisites
+
+- Install [Dumpling](/dumpling-overview.md) and [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md).
+- Set up [Data Migration (DM)](/dm/dm-overview.md)
```

Comment:
We can mention that these prerequisites are optional per "migration strategy"?

Reply:
OK, moved to L27